### ISSUE TITLE
Split vertical has incorrect hotkey hint in menu

### ISSUE THREAD

**Issue by flexponsive** [wez/wezterm#6115]
### What Operating System(s) are you seeing this problem on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Did you try the latest nightly build to see if the issue is better (or worse!) than your current version?

Yes, and I updated the version box above to show the version of the nightly that I tried

### Describe the bug

In the global menu under Shell > Split Vertical, it is indicated that the hotkey is CTRL+SHIFT+ALT+backtick, but the correct hotkey is not with quote not backtick.

Menu:

<img width="476" alt="image" src="https://github.com/user-attachments/assets/fed6a484-1b8c-4015-aadd-5175398c66ad">

Correct Key:

https://github.com/wez/wezterm/blob/30345b36d8a00fed347e4df5dadd83915a7693fb/docs/config/default-keys.md?plain=1#L79

### Configuration

no config

### Expected Behavior

The menu should show a quote symbol not backtick.

Presumably this is the line that should be changed: https://github.com/wez/wezterm/blob/30345b36d8a00fed347e4df5dadd83915a7693fb/wezterm-gui/src/commands.rs#L1460

**Comment by neogeographica** [comment]
Hmm that's kind of odd. Here's what that menu looks like for me, same wezterm version on macOS 13.6.3:

<img width="549" alt="Screenshot 2024-09-10 at 2 25 39 PM" src="https://github.com/user-attachments/assets/0aedfab5-c0a8-4379-ab43-e3d5d03a7913">

**Comment by flexponsive** [comment]
Very odd - I'm on an M1 MacBook Air, German language & region. macOS 14.4.1 (23E224)

**Comment by neogeographica** [comment]
Yeah, seems like a region thing right? And maybe also interacting with the whole area of physical keys vs keycodes or whatnot (that I'm not real familiar with).

I'm not clear on how that displayed key combo in the menu gets generated. In the CommandDef the key is specified as single-quote ```'```, which on my system shows as double-quote ```"``` and on yours as backtick ``` ` ```.

**Comment by wez** [comment]
Sounds like a weird interaction with macos's language/locale selection and how the key equivalent is represented in the menu bar.  Is this just a visual issue? Does the key assignment shown by `wezterm show-keys` work? Does the assignment shown in the menu also have that function?  Is that blocking some other assignment/action?

**Comment by flexponsive** [comment]
Here is the show-keys output:

```bash
$ wezterm show-keys | grep Vertical
        ALT | CTRL           "                  ->   SplitVertical(SpawnCommand domain=CurrentPaneDomain)
        SHIFT | ALT | CTRL   "                  ->   SplitVertical(SpawnCommand domain=CurrentPaneDomain)
        SHIFT | ALT | CTRL   '                  ->   SplitVertical(SpawnCommand domain=CurrentPaneDomain)
```

With my keyboard layout, `SHIFT | 2 -> "`.  So the difference between the first and second key map is not clear to me, but any way - pressing the physical keys `SHIFT | ALT | CTRL | 2` works to trigger SplitVertical. The single quote key map shown in the third row also works.

But the backtick (`SHIFT | ´ ` on my keyboard), as shown in the global menu, does not work.

So I guess this is just a purely visual issue. Ideally, the menu would show the single quote key map from the third row above when the keyboard layout is German, but this is most convenient key combination.

Thanks for this great terminal app!
